src/otx/core/model/entity/detection.py

```diff
@@ -27,6 +29,47 @@
 class OTXDetectionModel(OTXModel[DetBatchDataEntity, DetBatchPredEntity]):
     """Base class for the detection models used in OTX."""
 
+    def unpack_det_tiles(self, inputs: TileBatchDetDataEntity) -> DetBatchPredEntity:
+        """Unpack detection tiles.
+
+        Args:
+            inputs (TileBatchDetDataEntity): Tile batch data entity.
+
+        Returns:
+            DetBatchPredEntity: Merged detection prediction.
+        """
+        pred_entities = []
+        for tiles, tile_infos, bboxes, labels in zip(
+            inputs.batch_tiles,
+            inputs.batch_tile_infos,
+            inputs.bboxes,
+            inputs.labels,
+        ):
+            tile_preds: list[DetBatchPredEntity] = []
+            for tile, tile_info in zip(tiles, tile_infos):
+                tile_input = DetBatchDataEntity(
+                    batch_size=1,
+                    images=[tile],
+                    imgs_info=[tile_info],
+                    bboxes=[bboxes],
+                    labels=[labels],
+                )
```

Comment:
Can we introduce a function such as `unbind()` to `TileBatchDetDataEntity` to clean this?
```suggestion
        for det_batch_data_entity in inputs.unbind():
            prediction = self.forward(det_batch_data_entity)
            pred_entities.append(merge_detection_tiles(tile_preds))

...

class TileBatchDetDataEntity:
    def unbind(self) -> list[BatchDetDataEntity]:
        ...
```

Reply:
@vinnamkim Thanks for the input. I added an unbind method to tile-related entities.